tests/assets/ansible/roles/caasp/tasks/Suse.yaml

```diff
@@ -35,7 +41,7 @@
   zypper:
       name:
         - skuba
-        - kubernetes-client
+        - kubernetes-1.18-client
```

Comment:
Is this needed? Isn't `kubernetes-client` refering to the latest available version?

Reply:
I found it strange too, I would have expected a virtual package pointing to 1.18 as in SP1 we had it pointing to 1.17, but that did not seems to be the case.
I'll double check though

Reply:
```
rookcheck-bleon-eb0a-master-0:~ # zypper search kubernetes
Loading repository data...
Reading installed packages...

S | Name                    | Summary                               | Type
--+-------------------------+---------------------------------------+--------
  | kubernetes-1.18-client  | Kubernetes client tools               | package
  | kubernetes-1.18-kubeadm | Kubernetes kubeadm bootstrapping tool | package
  | kubernetes-1.18-kubelet | Kubernetes kubelet daemon             | package
rookcheck-bleon-eb0a-master-0:~ # zypper install kubernetes-client
Loading repository data...
Reading installed packages...
'kubernetes-client' not found in package names. Trying capabilities.
No provider of 'kubernetes-client' found.
Resolving package dependencies...

Nothing to do.
rookcheck-bleon-eb0a-master-0:~ #
```